snapcraft/commands/validation_sets.py

```diff
@@ -90,11 +90,12 @@ def fill_parser(self, parser: argparse.ArgumentParser) -> None:
         )
         parser.add_argument("account_id", metavar="account-id")
         parser.add_argument("set_name", metavar="set-name")
-        parser.add_argument("sequence", metavar="sequence")
+        parser.add_argument("sequence", metavar="sequence", type=int)
```

Comment:
This gives an earlier and better error message than previously:

```
snapcraft internal error: StoreValidationSetsError(<Response [400]>)
```

Reply:
Makes sense, though we should still catch that and convert it better. Not sure whether that's worthwhile right now or if we should just integrate it into a craft-store migration.

Reply:
I'm fine to do this when migrating to craft-store.  It's an uncommon error on an uncommon command, so it's not a high priority.